----------------------------------------------------------------------
Revision: 282605965d2a3714bb0ef0deece20fbadfeb56bc
Parent:   d31e3528a05a44cb37564f06484906872f10eb5d
Author:   yan12125@gmail.com
Date:     2019/06/14 09時36分27秒
Branch:   net.venge.monotone

Changelog: 

Another fix for Mac OS X <= 10.12

Changes against parent d31e3528a05a44cb37564f06484906872f10eb5d

  patched  src/platform.hh

============================================================
--- src/platform.hh	9276d8a314fbca9e78e8f3963b77331e094b1b30
+++ src/platform.hh	5550553cda41431417a90c7569ad31efea9d671b
@@ -15,6 +15,7 @@
 
 #include <cstdio>
 #include <ctime>
+#include <sys/types.h>
 
 void read_password(std::string const & prompt, char * buf, size_t bufsz);
 void get_system_flavour(std::string & ident);
